keyboards/playkbtw/pk60/keymaps/rfvizarra/keymap.c

```diff
@@ -0,0 +1,30 @@
+#include QMK_KEYBOARD_H
+
+#define _______ KC_TRNS
```

Comment:
When you use `#include QMK_KEYBOARD_H`, this is included already. You can take it out

```suggestion

```